src/TestSuite/Fixture/ChecksumTestFixture.php

```diff
@@ -15,43 +17,43 @@
  */
 class ChecksumTestFixture extends TestFixture
 {
+    /**
+     * List of table hashes
+     *
+     * @var array
+     */
+    public static $tableHashes = [];
```

Comment:
Does this property need to be public? Given the underscore prefix earlier, the "public" visibility might be a mistake and it should be "protected" instead.